sample/test-sample/frontend/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@eclipse-sirius/sirius-web",
```

Comment:
Should this use a different name than the "real" sirius-web app?

Reply:
Done